### PR TITLE
Adding endpoints for enqueuing jobs

### PR DESCRIPTION
**Pull request by camertron**
Current endpoints are hooks/github and hooks/transifex. New endpoints are hooks/github/enqueue and hooks/transifex/enqueue.

@lumoslabs/platform

### CODE REVIEW

File: queue/lib/txgh-queue/job.rb [hunk 1]
```diff
@@ -57,7 +59,15 @@ def handle_transifex_hook(project, repo, payload)
...
 
-      handler.execute
+      execute(handler)
+    end
+
+    def execute(handler)
+      if TxghQueue::Config.processing_enabled?
+        handler.execute
+      else
+        respond_with(200, 'Ok')
```
Comment (zvkemp) [inline queue/lib/txgh-queue/job.rb:69]: Does the github push require a 200 response?

**Review by zvkemp — COMMENTED** [review]
Suggestion:

remove the `queue_configured?` checks and provide a default config that represents an null queue (returns a 500, 'Queue not configured') error

**Comment by camertron** [comment]
@zvkemp great suggestion :)

File: queue/lib/txgh-queue/job.rb (hunk 1 above)
Comment (camertron) [reply queue/lib/txgh-queue/job.rb:69]: I think it just wants something in the 2 or 300s. Is there a more appropriate response code?

**Review by zvkemp — APPROVED** [review]
🆒

**Comment by camertron** [comment]
Thanks @zvkemp!

**Pull request merged**